code/src/components/Form.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+export const Form = ({ messageNew ,setMessageNew, onFormSubmit }) => {
+
+    const onMessageNewChange = (event) => {
+        setMessageNew(event.target.value)
+      }
+
+    return (
+        <>
+            <form className='form-container' onSubmit={onFormSubmit}>
+                <label tabIndex='0' htmlFor='newMessage'>What's making you happy right now?</label>
+                <textarea 
+                    id='newMessage'
+                    type='text'
+                    rows='5'
+                    value={messageNew}
+                    placeholder='Tell us all about it :)'
+                    onChange={onMessageNewChange}
+                    className = {messageNew.length > 140 ? 'textarea-invalid' : 'textarea-valid'}
```

Comment:
Great and descriptive naming of css classes here as always in your projects! :) As mentioned above, if using maxLength then the red color could start slightly before 140 characters, or it could be applied to the ```submit-charcount-container>p``` if you wanted it to be more discreet. Just a thought :)